BootloaderCommonPkg/Library/LiteVariableLib/LiteVariableLib.c

```diff
@@ -363,17 +365,23 @@ InternalGetVariable (
   UINT32                  VariableNameLen;
   UINT32                  VariableDataLen;
   UINTN                   DataSizeIn;
+  EFI_GUID                *VarGuid;
 
   if ((DataSize == NULL) || (VariableName == NULL)) {
     return EFI_INVALID_PARAMETER;
   }
 
+  VarGuid = VariableGuid;
+  if (VarGuid == NULL) {
+    VarGuid = &gZeroGuid;
```

Comment:
@gdong1 is it suggested to declare GUID somewhere in header as below? although actually I don't find any warning in build log, only find complain in VS code. thanks

extern EFI_GUID gZeroGuid

Reply:
EDK2 build should put this extern in the autogen.h since it is listed in the [Guids] section of the .inf file.

Reply:
That's true. the build tool will generate it. I think it is not necessary to have a header file if there is no GUID structure for this guid.